pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -179,9 +178,14 @@ type PDReplicationConfig struct {
 	// +optional
 	LocationLabels StringSlice `toml:"location-labels,omitempty" json:"location-labels,omitempty"`
 	// StrictlyMatchLabel strictly checks if the label of TiKV is matched with LocaltionLabels.
-	// Immutable, change should be made through pd-ctl after cluster creation
+	// Immutable, change should be made through pd-ctl after cluster creation.
+	// Imported from v3.1.0
 	// +optional
 	StrictlyMatchLabel *bool `toml:"strictly-match-label,omitempty" json:"strictly-match-label,string,omitempty"`
+
+	// When PlacementRules feature is enabled. MaxReplicas and LocationLabels are not uesd any more.
```

Comment:
```suggestion
	// When PlacementRules feature is enabled, MaxReplicas and LocationLabels are not used anymore.
```

Reply:
I think we should keep the note as same as pd